content/fast/preload-critical-assets/codelab-preload-web-fonts.md

```diff
@@ -0,0 +1,62 @@
+---
+page_type: glitch
+title: Preload web fonts to improve loading speed
+author: garima
+description: |
+  In this codelab, learn how to improve the performance of a page by preloading
+  web fonts.
+web_updated_on: 2018-12-06
+web_published_on: 2018-11-05
+glitch: web-dev-preload-webfont
+---
+
+This code lab shows you how to preload web fonts using rel="preload" to remove any flash of unstyled text (FOUT).
+
+
+## Measure
+
+First measure how the website performs before adding any optimizations.
+Firstly, measure when fonts are loaded on your site. Follow these steps to check.
```

Comment:
I think you can delete this line. Line 18 seems to cover the same information.

Reply:
Updated comments based on the suggestions